docs/pl/docs/features.md

```diff
@@ -0,0 +1,200 @@
+
+Ostatnia ankieta <abbr title="coroczna ankieta przeprowadza w środowisku programistów języka Python">Python developer survey</abbr> jasno wskazuje, że <a href="https://www.jetbrains.com/research/python-developers-survey-2017/#tools-and-features" class="external-link" target="_blank">najczęściej używaną funkcjonalnością jest autouzupełnianie w edytorze</a>.
+
+Cała struktura frameworku **FastAPI** jest na tym oparta. Autouzupełnianie działa wszędzie.
+
+Rzadko będziesz musiał wracać do dokumentacji.
+
+Oto, jak twój edytor może Ci pomóc:
+
+* <a href="https://code.visualstudio.com/" class="external-link" target="_blank">Visual Studio Code</a>:
+
+![wsparcie edytora](https://fastapi.tiangolo.com/img/vscode-completion.png)
+
+* <a href="https://www.jetbrains.com/pycharm/" class="external-link" target="_blank">PyCharm</a>:
+
+![wsparcie edytora](https://fastapi.tiangolo.com/img/pycharm-completion.png)
+
+Otrzymasz uzupełnienie nawet w miejscach, w których normalnie uzupełnienia nie ma. Na przykład klucz "price" w treści JSON (który mógł być zagnieżdżony), który pochodzi z zapytania.
+
+Koniec z wpisywaniem błędnych nazw kluczy, przechodzeniem tam i z powrotem w dokumentacji lub przewijaniem w górę i w dół, aby sprawdzić, czy w końcu użyłeś nazwy `username` czy `user_name`.
+
+### Zwięzłość
+
+Wszystko posiada sensowne **domyślne wartości**. Wszędzie znajdziesz opcjonalne konfiguracje. Wszystkie parametry możesz dostroić, aby zrobić to co potrzebujesz do zdefiniowania API.
+
+Ale domyślnie wszystko **"po prostu działa"**.
+
+### Walidacja
+
+* Walidacja większości (lub wszystkich?) **typów danych** Pythona, w tym:
+    * Obiektów JSON (`dict`).
+    * Tablic JSON (`list`) ze zdefiniowanym typem elementów.
+    * Pól tekstowych (`str`) z określeniem minimalnej i maksymalnej długości.
+    * Liczb (`int`, `float`) z wartościami minimalnymi, maksymalnymi, itp.
+
+* Walidacja bardziej egzotycznych typów danych, takich jak:
+    * URL.
+    * Email.
+    * UUID.
+    * ...i inne.
+
+Cała walidacja jest obsługiwana przez ugruntowaną i solidną bibliotekę **Pydantic**.
+
+### Bezpieczeństwo i uwierzytelnianie
+
+Bezpieczeństwo i uwierzytelnianie jest zintegrowane. Bez żadnych kompromisów z bazami czy modelami danych.
+
+Wszystkie schematy bezpieczeństwa zdefiniowane w OpenAPI, w tym:
+
+* Podstawowy protokół HTTP.
+* **OAuth2** (również z **tokenami JWT**). Sprawdź samouczek [OAuth2 with JWT](tutorial/security/oauth2-jwt.md){.internal-link target=_blank}.
+* Klucze API w:
+    * Nagłówkach.
+    * Parametrach zapytań.
+    * Ciasteczkach, itp.
+
+Plus wszystkie funkcje bezpieczeństwa Starlette (włączając w to **<abbr title="po angielsku: session cookies">ciasteczka sesyjne</abbr>**).
+
+Wszystko zbudowane jako narzędzia i komponenty wielokrotnego użytku, które można łatwo zintegrować z systemami, magazynami oraz bazami danych - relacyjnymi, NoSQL, itp.
+
+### Dependency Injection
```

Comment:
While I agree that the name Dependency Injection is so well known that most likely even non-english speaking programmers are likely to be familiar with it and the feature itself is a key feature of FastAPI, I still think that the term **Wstrzykiwanie Zależności** should be used here. It's well established in polish literature.

I would suggest going with:
### Wstrzykiwanie Zależności (Dependency Injection)

or the other way around:
### Dependency Injection (Wstrzykiwanie Zależności)

Reply:
I was thinking about it for a while, but then I noticed that in some other language versions of documentation it's translated, and in others it's not.
So I've decided to go with the same pattern as in spanish translation, which is using "Dependency Injection" as a header but in the next line is adding title for another occurrence, but with explaination in spanish:
![image](https://user-images.githubusercontent.com/50829834/190896943-0dbf7496-ded9-49e0-a5d5-5a67f5a1638e.png)

My reasoning for it was that "Dependency Injection" is definitely used more often in web development industry than "Wstrzykiwanie Zależności", but anyway, to not leaving it untranslated, I've provided translation for it in the next line.

But I agree with you, I'll go with just "Wstrzykiwanie Zależności" as a header, and english version of it in a tooltip (like on the screenshot, but "reversed").

Let me know if that will be fine.

Reply:
Yes, I guess that makes sense.